### ISSUE TITLE
repo sync of android-9.0 stuck at 98%

### ISSUE THREAD

**Issue by lucky-child** [omnirom/android#35]
I'm not able to repo sync the android-9.0 branch. I've tried on several machines with no success. All works fine until 98% then it's always stuck at:
Fetching projects:  98% (619/627) android_vendor_omni
I've also tried an overnight run without success.

**Comment by Aduser345** [comment]
Same here
pressing enter asks for password but doesnt do anythin after that

**Comment by threadreaper** [comment]
I have the same issue.

**Comment by Simonmicro** [comment]
Yeah, same here - pressing return brings up the HTTP basic auth of GitHub (or informs about an failed host key verification). Regardless of the version - **it just gets stuck and wont sync**!
But hey! Maybe it has something to do with the private Omni Repos? Maybe commenting...
```xml
  <!-- include registered repos -->
  <include name="omni-private.xml" />
```
from `.repo/manifests/omni-default.xml` helps? At least for me!

**Comment by vonmatterlorenzohorn** [comment]
> Yeah, same here - pressing return brings up the HTTP basic auth of GitHub (or informs about an failed host key verification). Regardless of the version - **it just gets stuck and wont sync**!
> But hey! Maybe it has something to do with the private Omni Repos? Maybe commenting...
> 
> ```
>   
>   <include name="omni-private.xml" />
> ```
> 
> from `.repo/manifests/omni-default.xml` helps? At least for me!

Thanks a LOT brother! This finally worked for my android 11 build. Now it is checking out only 721 projects instad of 728(it ignored the ones that are non-existent on github). I'm a noobie and this is my first rom, and I was considering just building lineage instead and clearing this. And yeah, the private memers should remove these projects or at least change the omni-default xml to not include the private stuff at all, because it hinders the progress of noobs like me. Again, thanks.

**Comment by 380583067** [comment]
> 

感谢分享，我删除了“.repo/manifests/omni-default.xml”里面的“  
  <include name="omni-private.xml" />”就成功了，不知道会不会影响正常编译和开机，我尝试去其他仓库找一找能否找到并克隆下来
